cgOne/blueprint.py

```diff
@@ -13,5 +13,60 @@ class Blueprint:
     """
 
     def __init__(self):
-        self.max_speed = 0
-        self.exhaust = 0
+    	self.max_speed = 0
+    	self.exhaust = 0
+    	self.type = 'Hatchback'
+    	self.exterior = 'Aluminum'
+    	self.interior = 'Fabric'
+    	self.make = 'Make'
+    	self.model = 'Model'
+    	self.colors = ['blue', 'red', 'green', 'white', 'black']
+    	self.fuel = 'Petrol'
+
+    	self.performance = 0
+    	self.emissions = 0
+    	self.utility = 0
+
+    #Assign a name for the make of the car
```

Comment:
So `docstrings` in Python are special: 

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

In particular, don't use `#`, use:

```python3
def set_make(self, make_name):
    """
    my docstring comment
    """

```

Then in a python repl, when you type `help(blueprints.set_make)` it'll show that comment. Nifty? :)